adagrams/game.py

```diff
@@ -1,11 +1,125 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+player_one = []
+pool_conversion = list(LETTER_POOL)
+
 def draw_letters():
-    pass
 
-def uses_available_letters(word, letter_bank):
-    pass
+    re_ordered_list = random.sample(pool_conversion, len(pool_conversion))
+    game_letters = re_ordered_list[:10]
+    return game_letters
+
+
+
+def uses_available_letters(word, letters):
+
+    matches = []
+    word = word.upper()
+    for letter in word:
+        if letter in letters:
+            if matches.count(letter) < letters.count(letter):
+                matches.append(letter)
+            elif matches.count(letter) == letters.count(letter):
+                continue
+    
+    if len(matches) == len(word):
+        return True
+    else:
+        return False
```

Comment:
Another approach for implementing this solution would be to use a dictionary.

You could create a dict that has letters in the `letters` list as keys and the values would be how many times the letters appears. So if `letters` was ["a", "a", "b"] then you could create dict like `avail_letters = {"a": 2, "b": 1} 

Then you could loop over every element in `word` and check if it is in the `avail_letters` dict. If the element in `word` was a key in `avail_letters` then you could decrement the value by 1. You'd use this logic to determine if `word` uses available letters from the list `letters`